{{ cookiecutter.package_name|slugify }}/pyproject.toml

```diff
@@ -48,6 +48,7 @@ streamlit = "^1.3.0"
 {%- endif %}
 {%- if cookiecutter.with_typer_cli|int %}
 typer = "^0.4.0"
+click = "<=8.0.4" # pinned to work around https://github.com/tiangolo/typer/issues/377, change when that is fixed
```

Comment:
Suggestion: let's give Typer until the end of today to fix this with a v0.4.1 before we merge this PR. (Then we don't need to rebase the other PRs).

Reply:
That's a good idea. Let's hold off for today then!

Reply:
Update: will close this PR since Typer was updated today.